setup.py

```diff
@@ -71,10 +69,10 @@
     },
     install_requires=[
         'numpy',
-        'tensorflow>=1.13,<1.14',  # Must be on piwheels
+        'tensorflow-gpu==2.2.0rc', # This should be changed for 2.2.0 when it's released.
         'sonopy',
         'pyaudio',
-        'keras<=2.1.5',
+        'keras>2.1.5',
```

Comment:
I think we can remove the `keras` dependency altogether since you've transitioned to `tensorflow.keras`, but I could be wrong.

Reply:
We should still keep Keras, as training relies on the native module and not tf.keras.

For an explanation, see:
https://github.com/MycroftAI/mycroft-precise/pull/141#discussion_r403766075

Reply:
This is no longer the case.
@MatthewScholefield your PR has been merged and the repository now uses only tf.keras